core/context.go

```diff
@@ -317,3 +317,17 @@ func contains(list []string, item string) bool {
 	}
 	return false
 }
+
+func anyInSlice(a, b []string) bool {
+    m := make(map[string]struct{}, len(a))
+    for _, v := range a {
+        m[v] = struct{}{}
+    }
+
+    for _, v := range b {
+        if _, ok := m[v]; ok {
+            return true
+        }
+    }
+    return false
+}
```

Comment:
This seems like a very strange way to check if elements in one slice are in another slice. Wouldn't it make more sense to iterate over the `wheelGroup` slice, and check if each entry is in the user's groups?

Reply:
It's theoretically faster but two nested for loops is fine here as the length of the slices is so small